Libs/MRML/Core/vtkMRMLAbstractViewNode.h

```diff
@@ -358,6 +358,10 @@ class VTK_MRML_EXPORT vtkMRMLAbstractViewNode
   /// Labels of coordinate system axes
   vtkSmartPointer<vtkStringArray> AxisLabels;
 
+  ///
+  /// Define if the view is mapped in the current layout with a Qwidget.
```

Comment:
MRML is not aware of Qt.

```suggestion
  /// Define if the view is displayed in the current layout.
```